apps/condo/domains/ticket/constants/statusTransitions.js

```diff
@@ -0,0 +1,24 @@
+/*
+* refs to: https://doma.atlassian.net/browse/SBERDOMA-97
+* */
+
+const STATUS_IDS = {
+    OPEN: '6ef3abc4-022f-481b-90fb-8430345ebfc2',
+    IN_PROGRESS: 'aa5ed9c2-90ca-4042-8194-d3ed23cb7919',
+    DECLINED: 'f0fa0093-8d86-4e69-ae1a-70a2914da82f',
+    COMPLETED: '5b9decd7-792c-42bb-b16d-822142fd2d69',
+    DEFERRED: 'c14a58e0-6b5d-4ec2-b91c-980a90509c7f',
+}
+
+const DEFAULT_STATUS_TRANSITIONS = {
+    'dv':1,
+    [STATUS_IDS.OPEN]: [STATUS_IDS.IN_PROGRESS, STATUS_IDS.DECLINED],
+    [STATUS_IDS.IN_PROGRESS]: [STATUS_IDS.DEFERRED, STATUS_IDS.COMPLETED, STATUS_IDS.OPEN],
+    [STATUS_IDS.COMPLETED]: [STATUS_IDS.OPEN], // TODO(Dimitreee): ADD relation to 'closed' status
+    [STATUS_IDS.DECLINED]: [],
+    [STATUS_IDS.DEFERRED]: [STATUS_IDS.OPEN],
+}
+
+module.exports = {
+    DEFAULT_STATUS_TRANSITIONS,
```

Comment:
I whin it should be different for different languages.
There are Russian language statuses.

Reply:
we have `organization.language`

Reply:
https://doma.atlassian.net/browse/SBERDOMA-118